packages/fxa-graphql-api/src/lib/server.ts

```diff
@@ -39,25 +58,42 @@ export async function createServer(
   const schema = await TypeGraphQL.buildSchema({
     container: Container,
     resolvers: [AccountResolver],
+    validate: false,
```

Comment:
This is only needed when we're using the `class-validator` package to validate input beyond the basic GraphQL type declarations.